src/app/settings/preferences/preferences.component.ts

```diff
@@ -48,7 +48,10 @@ export class PreferencesComponent implements AfterContentInit {
     private _submissionService: VbsSubmissionService,
     private _notificationService: NotificationService
   ) {
-    this._configService.configAsObservable.subscribe(c => this._config = c)
+    this._configService.configAsObservable.subscribe(c => {
+      this._config = c
+      this.maxLength = c._config.query.temporal_max_length
```

Comment:
I don't fully understand why this is a property on this level? For convenience? @silvanheller